tests/integration/140.command.sites.test.js

```diff
@@ -179,3 +180,29 @@ test.serial('should return an array of templates with name, source code url and
     ])
   })
 })
+
+test.serial('should throw error when name flag is incorrect', async (t) => {
+  await withMockApi(routes, async ({ apiUrl }) => {
+    Object.defineProperty(process, 'env', {
+      value: {
+        NETLIFY_API_URL: apiUrl,
+        NETLIFY_AUTH_TOKEN: 'fake-token',
+      },
+    })
+
+    const program = new BaseCommand('netlify')
+
+    createSitesCreateCommand(program)
+
+    const missingNameError = await t.throwsAsync(async () => {
+      await program.parseAsync(['', '', 'sites:create', '--name'])
+    })
+    t.truthy(missingNameError.message.includes('Please specify site name'))
```

Comment:
I do not think we need this test anymore, once we make the arguments mandatory.

As a sidenote: If you write tests it might be better to split the two tests you created into two test functions. Because right now when the first one fails the second one won't be executed at all.